.github/workflows/release.yml

```diff
@@ -180,6 +180,25 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.ORG_GORELEASER_GITHUB_TOKEN }}
           COSIGN_PWD: ${{ secrets.ORG_COSIGN_PWD }}
+      # Create artifact bundle and upload to release
+      - name: Create artifact archive
+        run : |
+          mkdir artifacts
+          cp ./scripts/install/*.sh ./artifacts
+          cp ./observiq-otel-collector.msi/observiq-otel-collector.msi ./artifacts
+          cp ./observiq-otel-collector-x86.msi/observiq-otel-collector-x86.msi ./artifacts
```

Comment:
I didn't know these were nested, makes me wonder how the glob we have in goreleaser is working at all!

Reply:
I also wondered that because at first I copied them as goreleaser did and got an error.